ReactiveObjC/RACSignal+Operations.h

```diff
@@ -14,7 +14,7 @@
 @class RACMulticastConnection<__covariant ValueType>;
 @class RACScheduler;
 @class RACSequence<__covariant ValueType>;
-@class RACSubject;
+@class RACSubject<__covariant ValueType>;
```

Comment:
You seem to be supposing that I've actually read up on any of these things. ;)

Why does being both a source and a sink confine an object to invariance? Shouldn't a `RACSubject<NSArray *>` also allow one to send `NSMutableArray`s? I think I'm missing something here.